src/validation.cpp

```diff
@@ -4706,6 +4706,11 @@ bool Chainstate::LoadChainTip()
               m_chain.Height(),
               FormatISO8601DateTime(tip->GetBlockTime()),
               GuessVerificationProgress(m_chainman.GetParams().TxData(), tip));
+
+    // Ensure KernelNotifications m_tip_block is set even if no new block arrives.
+    // Ignoring return value for now.
+    (void)m_chainman.GetNotifications().blockTip(GetSynchronizationState(/*init=*/true, m_chainman.m_blockman.m_blockfiles_indexed), *pindex);
```

Comment:
nit: at this point it may make more sense to just remove the `[[nodiscard]]` from `blockTip` instead

Reply:
Maybe, or we could do something with the return value. I'm not familiar enough with init code and kernel notifications to have a strong opinion on this. cc @ryanofsky, @TheCharlatan

Reply:
Just ignoring the return value is probably the best thing for now since this function and its callers return bool. After #29700 it could more easily bubble up the interrupted status. The kernel API is designed for a better world where operations can be more easily interrupted but not everything is interruptible not now.